torch/overrides.py

```diff
@@ -1924,8 +1924,9 @@ def _push_mode(mode):
 
 
 def _pop_mode():
-    old = _pop_torch_function_stack()
-    return old
+    if _len_torch_function_stack() > 0:
```

Comment:
Hmm. silently returning None here is probably ok? But what are we doing that causes us to need to do this?